Gemfile

```diff
@@ -1,49 +1,41 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.7.1'
-
-# Note that mysql2 0.4.X doesn't work with Rails. 4.2.X yet.
-gem 'mysql2', '~> 0.3.20'
-
-gem 'plek', '~> 1.11.0'
-gem 'airbrake', '~> 4.3.5'
-
-gem 'gds-sso', '~> 11.0.0'
-gem 'gds-api-adapters', '~> 30.2.0'
-
-gem 'govuk_admin_template', '~> 4.1'
-gem 'generic_form_builder', '~> 0.13.0'
-gem 'aasm', '~> 4.3.0'
+gem 'rails', '~> 5.0'
 
+gem 'aasm', '~> 4.0'
+gem 'airbrake', '~> 6.0'
```

Comment:
5.5 is working in Publishing API – https://github.com/alphagov/publishing-api/blob/master/config/initializers/airbrake.rb